src/pages/settings/Subscription/RequestEarlyCancellationPage/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import FeedbackSurvey from '@components/FeedbackSurvey';
+import FixedFooter from '@components/FixedFooter';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useCancellationType from '@hooks/useCancellationType';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as Report from '@userActions/Report';
+import * as Subscription from '@userActions/Subscription';
+import type {FeedbackSurveyOptionID} from '@src/CONST';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function RequestEarlyCancellationPage() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const cancellationType = useCancellationType();
+
+    const handleSubmit = (cancellationReason: FeedbackSurveyOptionID, cancellationNote = '') => {
+        Subscription.cancelBillingSubscription(cancellationReason, cancellationNote);
+    };
+
+    const acknowledgmentText = useMemo(
+        () => (
+            <Text>
+                {translate('subscription.requestEarlyCancellation.acknowledgement.part1')}
+                <TextLink href={CONST.TERMS_URL}>{translate('subscription.requestEarlyCancellation.acknowledgement.link')}</TextLink>
+                {translate('subscription.requestEarlyCancellation.acknowledgement.part2')}
+            </Text>
+        ),
+        [translate],
+    );
+
+    const manualCancellationContent = useMemo(
+        () => (
+            <View style={[styles.flexGrow1, styles.justifyContentBetween, styles.mh5]}>
+                <View>
+                    <Text style={styles.textHeadline}>{translate('subscription.requestEarlyCancellation.requestSubmitted.title')}</Text>
+                    <Text style={[styles.mt1, styles.textNormalThemeText]}>
+                        {translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.part1')}
+                        <TextLink onPress={() => Report.navigateToConciergeChat()}>{translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.link')}</TextLink>
+                        {translate('subscription.requestEarlyCancellation.requestSubmitted.subtitle.part2')}
+                    </Text>
+                </View>
+                <FixedFooter style={styles.ph0}>
+                    <Button
+                        success
+                        text={translate('common.done')}
+                        onPress={() => Navigation.goBack()}
```

Comment:
what happens when we hit this page in the address bar.  Do we need to add a fallback route?

Reply:
> display "not found" if accessed improperly, do you think we should add it

@rlinoz Any thoughts?